python/ray/data/dataset.py

```diff
@@ -2871,7 +2895,13 @@ def to_tf(
 
         Returns:
             A ``tf.data.Dataset`` that yields inputs and targets.
-        """
+
+        .. seealso::
+
+            :meth:`~ray.data.Dataset.iter_tf_batches`
+                Call this method if you need more flexibility.
```

Comment:
Nice, good `seealso`, we should have more of these kind of related cross-references.